test/smoke/src/areas/workbench/viewlet.ts

```diff
@@ -12,6 +12,6 @@ export abstract class Viewlet {
 	constructor(protected code: Code) { }
```

Comment:
Great that you updated the smoke test. Is the smoke test green after all these changes?

Reply:
@isidorn We have PR integration https://vscode.visualstudio.com/VSCode/_build/results?buildId=3076&view=logs